source/develop/release-management/features/network/networkfilterparameters.html.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Network Filter Parameters
+category: feature
+authors: dholler
+wiki_category: Feature
+wiki_title: Network/NetworkFilterParameters
+wiki_revision_count: 0
+wiki_last_updated: 2016-11-08
+feature_name: Network Filter Parameters
+feature_modules: engine,vdsm
+feature_status: In Development
+---
+
+# Network Filter Parameters
+
+## Summary
+
+The attachment of a predefined [network filters by vNIC profiles to VMs is already available][2],
+but the user is not yet able to set [parameters][3] of the network filters.
+There are scenarios, e.g. [Bug 1366905][4], in which there is a need to set some of the parameters of the network filters.
+This feature enables the user to set the values of the parameters explicitly. 
+
+## Owner
+
+*   Name: [Dominik Holler](User:dominikholler)
+
+*   Email: <dholler@redhat.com>
+
+## Detailed Description
+
+Expand on the summary, if appropriate. A couple sentences suffices to explain the goal, but the more details you can provide the better.
+
+## Benefit to oVirt
+
+This feature enables the usages of oVirt in scenarios, in which the user has to set parameters, without forcing the user to edit configuration files on the hosts manually.
+
+### User work-flows
+
+To use network filter parameters, the following workflow is expected:
+
+1. The user configures a vNIC Profile to apply a network filter, which depends on parameters.
+
+2. The user configures a logical network, to apply this vNIC Profile.
+
+3. The user connects a virtual machines network interface to this logical network.
+
+4. The user creates network filter parameters for this network interface.
+
+### User Experience
+
+Describe user experience related issues. For example: We need a wizard for ...., the behaviour is different in the UI because ....., etc. GUI mockups should also be added here to make it more clear
```

Comment:
I added a UI mockup. There is added a section for the Network Filter Parameters. Because the the UI has no knowledge about the filter, it provides the ability to create network filter parameters for all network filters.